src/tags/TagChip.tsx

```diff
@@ -0,0 +1,36 @@
+import { Chip } from '@mui/material';
+import { useState } from 'react';
+import { Tag } from '../types';
+import { TagEditModal } from './TagEditModal';
+
+type TagChipProps = {
+    tag: Tag;
+
+    onUnlink: () => Promise<void>;
+};
+
+export function TagChip({ tag, onUnlink }: TagChipProps) {
```

Comment:
**question**: why call that `onUnlink`? :thinking:

Reply:
I did not want to call this onDelete, since delete naming will be used for deletion once it is implemented